mission_summary/mission_summary.py

```diff
@@ -0,0 +1,209 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+from glidertest import tools, plots
+from ioos_qc import qartod
+import cartopy.crs as ccrs
+from matplotlib.font_manager import FontProperties
+
+configs = {
+    "TEMP": {"gross_range_test": {
+        "suspect_span": [0, 30],
+        "fail_span": [-2.5, 40],
+    },
+        "spike_test": {"suspect_threshold": 2.0, "fail_threshold": 6.0},
+        "location_test": {"bbox": [10, 50, 25, 70]},
+    },
+    "PSAL": {"gross_range_test": {"suspect_span": [5, 38], "fail_span": [2, 41]},
+             "spike_test": {"suspect_threshold": 0.3, "fail_threshold": 0.9},
+             "location_test": {"bbox": [10, 50, 25, 70]},
+             },
+    "DOXY": {"gross_range_test": {
+        "suspect_span": [0, 350],
+        "fail_span": [0, 500],
+    },
+        "spike_test": {"suspect_threshold": 10, "fail_threshold": 50},
+        "location_test": {"bbox": [10, 50, 25, 70]},
+    },
+    "CHLA": {"gross_range_test": {
+        "suspect_span": [0, 15],
+        "fail_span": [-1, 20],
+    },
+        "spike_test": {"suspect_threshold": 1, "fail_threshold": 5},
+        "location_test": {"bbox": [10, 50, 25, 70]},
+    },
+}
+
+
+def qc_checks(ds, var='TEMP'):
+    gr = tools.compute_global_range(ds, var=var, min_val=configs[var]['gross_range_test']['suspect_span'][0],
+                                    max_val=configs[var]['gross_range_test']['suspect_span'][1])
+    spike = qartod.spike_test(ds[var], suspect_threshold=configs[var]['spike_test']['suspect_threshold'],
+                              fail_threshold=configs[var]['spike_test']['fail_threshold'], method="average")
+    flat = qartod.flat_line_test(ds[var], ds.TIME, 1, 3, 0.001)
+
+    df, diff, err, rms = tools.compute_hyst_stat(ds, var=var, v_res=1)
```

Comment:
You aren't using several of these variables like diff, rms. Instead of assigning variables, you can discard them with __ i.e.:

`    __, __, err, __ = tools.compute_hyst_stat(ds, var=var, v_res=1)`

This lets others using it know that you are only using err from this functions. So in the case of e.g. a future refactor, we don't need to look after df